src/api/airdropEligibility.js

```diff
@@ -0,0 +1,11 @@
+const { get } = require('request-promise')
+
+module.exports = async (dvf, ethAddress) => {
+  if (ethAddress) {
```

Comment:
Sorry for the late comment, but I would also encourage you to perform the arguments validation in a similar way vs. the other files. For example https://github.com/DeversiFi/dvf-client-js/blob/master/src/api/getRegistrationStatuses.js is very similar (one "data" or "args" param and use of Joi to validate them and return the appropriate error automatically)

Reply:
Nice, thanks!